plugins/base/src/main/kotlin/renderers/DefaultRenderer.kt

```diff
@@ -140,7 +157,7 @@ abstract class DefaultRenderer<T>(
     }
 
     open suspend fun renderPage(page: PageNode) {
-        val path by lazy { locationProvider.resolve(page, skipExtension = true) }
+        val path by lazy { locationProvider.resolve(page, skipExtension = true)!! }
```

Comment:
Would you mind replacing the bangs with an error message? 🙄

Reply:
Forgot about them completely. Sure!